src/app/(pages)/projects/page.tsx

```diff
@@ -0,0 +1,16 @@
+"use client";
+ import { useQuery} from 'react-query';
+ import { getAllProjects } from '@/app/Service/ProjectServices';
+import ResponsiveDrawer from '@/app/components/drawer/Drawer';
+
+
+function page() {
+   const { data: Allprojects} = useQuery('Allprojects', getAllProjects);
+  return (
+    <>
+    <ResponsiveDrawer projects={Allprojects}/>
```

Comment:
why you add responsive drawer here and in layout 
in layout is the good practice you should remove it from here